docs/EN_US/ECLLanguageReference/ECLR_mods/Basics-Constants.xml

```diff
@@ -236,81 +175,74 @@ MyReal2 := 1.0e1;  // value of MyReal2 is the REAL value 10.0
                   <primary>__ECL_VERSION__</primary>
                 </indexterm></emphasis></entry>
 
-              <entry>A STRING containing the
-              value of the platform version. For example, '6.4.0'</entry>
+              <entry>A STRING containing the value of the platform version.
+              For example, '6.4.0'</entry>
             </row>
 
             <row>
-              <entry><emphasis>__ECL_VERSION_MAJOR__
-              <indexterm>
+              <entry><emphasis>__ECL_VERSION_MAJOR__ <indexterm>
                   <primary>__ECL_VERSION_MAJOR__</primary>
                 </indexterm></emphasis></entry>
 
-              <entry>An INTEGER containing
-              the value of the major portion of the platform version. For
-              example, '6'</entry>
+              <entry>An INTEGER containing the value of the major portion of
+              the platform version. For example, '6'</entry>
             </row>
 
             <row>
               <entry><emphasis>__ECL_VERSION_MINOR__<indexterm>
                   <primary>__ECL_VERSION_MINOR__</primary>
                 </indexterm></emphasis></entry>
 
-              <entry>An INTEGER containing
-              the value of the minor portion of the platform version. For
-              example, '4'</entry>
+              <entry>An INTEGER containing the value of the minor portion of
+              the platform version. For example, '4'</entry>
             </row>
 
             <row>
               <entry><emphasis>__ECL_LEGACY_MODE__<indexterm>
                   <primary>__ECL_LEGACY_MODE__</primary>
                 </indexterm></emphasis></entry>
 
-              <entry>A BOOLEAN value
-              indicating if it is being compiled with legacy IMPORT
-              semantics.</entry>
+              <entry>A BOOLEAN value indicating if it is being compiled with
+              legacy IMPORT semantics.</entry>
             </row>
 
             <row>
               <entry><emphasis>__OS__<indexterm>
                   <primary>__OS__</primary>
                 </indexterm></emphasis></entry>
 
-              <entry>A STRING indicating the
-              operating system to which it is being compiled. (Available
-              values are: 'windows' or 'linux')</entry>
+              <entry>A STRING indicating the operating system to which it is
+              being compiled. (Available values are: 'windows' , 'macos', or
```

Comment:
Can we change 'available' to 'possible'?  That makes more sense to me and lines up with the description for TARGET_PLATFORM.